netci.groovy

```diff
@@ -107,13 +106,17 @@ def static calculateBuildCommands(def os, def configuration, def scenario, def i
 
     if (os == 'Windows_NT') {
         // Calculate the build commands
-        buildCommands += "build.cmd ${lowercaseConfiguration}"
+        buildCommands += "build.cmd ${lowercaseConfiguration} skiptests"
 
-        // Calculate the test commands
-        buildCommands += "tests\\runtest.cmd ${configuration} /multimodule"
         if (configuration == 'Debug')
         {
             if (scenario == 'coreclr'){
+                // Run simple tests and multimodule tests only under CoreCLR mode
```

Comment:
Could you pull the simple and multimodule invocations out of here (maybe into a separate `if coreclr` right after the `build.cmd` invocation? We would like these to also run in Release configurations and with this it would only be on Debug.